src/Sulu/Bundle/MediaBundle/Media/FormatManager/FormatManager.php

```diff
@@ -527,6 +542,7 @@ protected function getMediaData($media)
                     $fileName = $fileVersion->getName();
                     $storageOptions = $fileVersion->getStorageOptions();
```

Comment:
I understand. In this case it makes sense, that the storages handle the json, decode, etc.

But the problem i mean, is that the storages only accept the encoded string, so a Storage(Manager) user can not know the the format the storage need. It's complicated, when the storage wanna accept an array (for usability), but how should the `FileVersion` entity know how to decode the storage options :fearful: